avm/ptn/azd/ml-project/main.json

```diff
@@ -0,0 +1,7427 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.28.1.47646",
```

Comment:
Not mandatory, just a heads up that a newer version of bicep is available, in case you'd like to upgrade it